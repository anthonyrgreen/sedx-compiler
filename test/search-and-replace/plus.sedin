word0 word1word1 word2word2word2