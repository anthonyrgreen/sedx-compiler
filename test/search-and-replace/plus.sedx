let
  myPlus := plus('word', anyOf('0-9'))
match
${capture(myPlus()) as words0} ${capture(myPlus()) as words1} ${capture(myPlus()) as words2}
substitute
${words2} ${words1} ${words0}