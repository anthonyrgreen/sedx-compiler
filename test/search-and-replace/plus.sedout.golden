word2word2word2 word1word1 word0